pycwr/io/WSR98DFile.py

```diff
@@ -1,12 +1,28 @@
 # -*- coding: utf-8 -*-
+import os
 import numpy as np
+from numpy.ctypeslib import ndpointer
 from .BaseDataProtocol.WSR98DProtocol import dtype_98D
 from .util import _prepare_for_read, _unpack_from_buf, julian2date_SEC, make_time_unit_str
 from ..core.NRadar import PRD
 from ..configure.pyart_config import get_metadata, get_fillvalue
 from ..configure.default_config import CINRAD_field_mapping
 from ..core.PyartRadar import Radar
 from netCDF4 import date2num
+import ctypes
+from ctypes import c_int, c_short, c_long, c_float, c_double, c_char, c_ubyte, c_char_p
+path0 = os.path.split(os.path.realpath(__file__))[0]
+
+dll_name = "subc_read_WSR98D"
+fullpath_dll = f"{path0}/lib_{dll_name}.so"
+if not os.path.exists(fullpath_dll):
+    cmd = f"gcc -fPIC -shared -O2 {path0}/{dll_name}.c -o {fullpath_dll}"
```

Comment:
看起来处理速度改善比较明显，可以考虑merge进来。但是有一点可能需要进一步测试：windows默认似乎是没有gcc的，此处调用系统命令进行编译链接的话是否会引入新的依赖，应该是需要测试并解决的。
```suggestion
    cmd = f"gcc -fPIC -shared -O2 {path0}/{dll_name}.c -o {fullpath_dll}"
```
@Morlyliu 请问是否测试过windows系统的运行情况？